### DISCUSSION TITLE
Project 64 Video Plugin: Aspect Ratio "Adjust Game To Fit"

### DISCUSSION THREAD

**Discussion by PimpUigi** [project64/project64#2266] [Ideas]
Semi feature request, it would be extremely helpful if the Project 64 Video Plugin wasn't just limited to 16:9 (anamorphic), 4:3 and stretch.
It would be nice if it could have a setting that expands the viewable game area to widescreen, like GlideN64's "Try to adjust game to fit"

Is this something that would be inherently difficult to add to the PJ 64 video plugin??

**Comment by oddMLan** [comment]
PJ64 Video it's a rebranded version of Glide64 which it's no longer maintained. I honestly don't see it happening.
EDIT: And GLideN64 will become default in the next release of PJ64

**Comment by dftf-stu** [comment]
@PimpUigi: I use the "Project64 Video" plug-in, and have such an option -- but it's usually hidden by-default.

Do the following:
1. Click the **Options** menu, then go to **Configuration**
2. Un-tick "**Hide advanced settings**", then click **[OK]**, then fully-close the _Project64_ app
3. Launch _Project64_ again (bring it back up on-screen after closing it)
4. Click the **Options** menu, then go to **Graphics Settings**
5. You should now have two tabs at the top: "Basic Settings" and "Emulation Settings"
6. Click the "Emulation Settings" tab
7. You should then see an "Aspect ratio" dropdown where you can change this:
![AspectRatio](https://user-images.githubusercontent.com/83965343/183104438-6a3031c0-217e-4968-8894-ed87e5dc4a8a.png)

If you do not see that option, you might need to download a newer version of _Project64_ from [here](https://www.pj64-emu.com/public-releases).

Does this solve your issue?
